src/plugins/dataflow-tool/components/dataflow-tool.tsx

```diff
@@ -31,7 +35,9 @@ export default class DataflowToolComponent extends BaseComponent<IProps, IState>
 
   public static tileHandlesSelection = true;
 
-  public state: IState = {};
+  public state: IState = {
+    isEditingTitle: false
+  };
```

Comment:
We don't actually seem to be using this state variable anywhere. It's declared and it's set and cleared but there doesn't seem to be any other reference to it, unless it's simply being used as a means to trigger a re-render.

Reply:
I put it in basically copying what was in place on `image-tool.tsx` and `geometry-content.tsx`. It looks like at the moment the `renderTableLinkButton()` in `geometry-content.tsx` needs to check to make sure the title is not being edited ahead of rendering the link button.  I could leave it in place in case we end up building anything similar on the Dataflow tile?

Reply:
I would take it out, then. It's easy enough to add back in if the need arises.